recipes/default.rb

```diff
@@ -24,7 +24,7 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-include_recipe 'python'
+python_runtime 'beaver'
```

Comment:
May be it's better to remove this explicit declaration of `python_runtime` as well as its mentioning in lines 50 and 71?

Docs of `poise-python` says that it will work with the recent declared runtime:
>python – Name of the python_runtime resource to use. If not specified, the most recently declared python_runtime will be used. Can also be set to the full path to a python binary.

So I guess it makes possible to configure runtime in wrapper cookbook. @yakara-ltd What do you think about it?